### DISCUSSION TITLE
Where to specify cursor hiding time?

### DISCUSSION THREAD

**Discussion by HubKing** [tomasklaen/uosc#769] [Q&A] [answered]
I set `autohide=yes` to hide the UI when the cursor gets hidden. The problem is that the cursor gets hidden to early. It seems like within 1 or 2 seconds. I want to change it to a longer value like 5 seconds.

I set `hidetimeout=5000` in mpv.conf, but that didn't work. Google search said it needs to be in osc.conf. But I don't have osc.conf. I tried adding it in uosc.conf, but that didn't work. Also tried creating a new osc.conf file in both the .config/mpv and the script-opts, and added hidetimeout=5000 there, but it did not work either.

**Comment by tomasklaen** [comment, marked as answer]
It's [`cursor-autohide`](https://mpv.io/manual/master/#options-cursor-autohide). Belongs in `mpv.conf`.
